### PR TITLE
Collapsed Text Indicator/Ellipsis Now has a dedicated dark-mode LESS style.

### PR DESCRIPTION
**Pull request by K0Stek122**
## Description

Fixes the not-so-good looking ellipsis in Dark Mode as per [issue 5204](https://github.com/Zettlr/Zettlr/issues/5204). It specifies in the dark mode LESS section to change the specific colors.
## Changes

No changes to existing code, added a few LESS lines.

## Additional information

The color scheme used removes the border that is in light mode. It looks a bit different than in the light mode but i ultimately decided that it looks better like that in dark mode. If this is a problem please say so and i'll tweak it.

Light Mode Unchanged:
image

Dark Mode Before:
image

Dark Mode After:
image

Tested on: Fedora Linux 39 (Xfce) x86_64

### CODE REVIEW

File: source/win-main/MainEditor.vue [hunk 1]
```diff
@@ -704,9 +704,16 @@ function maybeHighlightSearchResults (): void {
   }
 }
 
+//These are styles specific to dark-style.
```
Comment (kyaso) [inline source/win-main/MainEditor.vue:707]: We can omit this comment (in the next line, we have `body.dark`, so it should be pretty clear that this applies to dark mode).

File: source/win-main/MainEditor.vue [hunk 2]
```diff
@@ -704,9 +704,16 @@ function maybeHighlightSearchResults (): void {
...
 body.dark .main-editor-wrapper {
   background-color: rgba(20, 20, 30, 1);
   .CodeMirror .CodeMirror-gutters { background-color: rgba(20, 20, 30, 1); }
+
+  //Three dots (...) When a header is folder:
```
Comment (kyaso) [inline source/win-main/MainEditor.vue:712]:
```suggestion
  // Ellipsis (...) when a header is folded
```

File: source/win-main/MainEditor.vue [hunk 3]
```diff
@@ -704,9 +704,16 @@ function maybeHighlightSearchResults (): void {
...
+  .cm-foldPlaceholder{
+        background-color: rgb(20, 20, 30);
```
Comment (kyaso) [inline source/win-main/MainEditor.vue:714]: We should fix the indentation here. Instead of 4 spaces, we should use 2 spaces.

**Review by kyaso — COMMENTED** [review]
Hey, thanks for your contribution!

I have just a few style-related remarks.

**Comment by K0Stek122** [comment]
All done.

**Comment by nathanlesage** [comment]
@kyaso What's your opinion, can we merge?

**Comment by kyaso** [comment]
LGTM, we can merge

**Comment by nathanlesage** [comment]
Wonderful, thanks a bunch! Merging now…

**Pull request merged**